setup.py

```diff
@@ -43,22 +44,20 @@
 )  # circumventing esmpy/xesmf installation issues; use conda
 # after complete is set, add in test
 extras_require["test"] = [
+    "netcdf4",
```

Comment:
Should this be replaced with h5netcdf?

Reply:
Locally it works with netcdf but I will add a comment here